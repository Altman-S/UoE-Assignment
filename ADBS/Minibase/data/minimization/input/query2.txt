Q(x) :- R(x, y), R(x, 4), R('ADBS', 4), R(u, 12), R(u, v), S('ADBS', 12, 7)