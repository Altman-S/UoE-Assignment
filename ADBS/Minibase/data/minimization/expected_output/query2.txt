Q(x) :- R(x, 4), R('ADBS', 4), R(u, 12), S('ADBS', 12, 7)